domain/network/state/types.go

```diff
@@ -705,6 +705,7 @@ func dmlToNetAddr(addr getIpAddress, deviceName string) network.NetAddr {
 		Scope:            corenetwork.Scope(addr.Scope),
 		IsSecondary:      addr.IsSecondary,
 		IsShadow:         addr.IsShadow,
+		Space:            addr.Space,
```

Comment:
Please rename these methods. `netAddrToDML` makes sense because we are converting an _incoming_ structure to one used in **D**ata **M**anipulation **L**anguage queries. There is no symmetrical opposite.